### PR TITLE
Include NotesFeed component

### PR DESCRIPTION
**Pull request by bernardodiasc**
screen recording 2018-05-24 at 03 38 pm

### CODE REVIEW

**Comment by bernardodiasc** [comment]
This PR is ready for Review. 👍

File: src/components/ui/NotesFeed.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+          )
+        ),
+        this.showViewMore((notes && notes.length) || 0) && (
+          <Button
+            outlined
+            block
+            color='silver'
+            onClick={this.viewMore}
+            className={buttonClass}
+          >
```
Comment (slawekkolodziej) [inline src/components/ui/NotesFeed.js]: Let's add `type='button'` to prevent submitting forms

**Comment by bernardodiasc** [comment]
@slawekkolodziej good call, I updated the code with your suggestion 👍

File: src/components/ui/Loader.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+// @flow
+
+import { PureComponent } from 'react'
+import elem from '../../utils/elem'
+
+type Props = {}
```
Comment (nicksp) [inline src/components/ui/Loader.js]: No need to define empty Props. Besides such simple component as Loader is better to write in a functional form.

**Pull request merged**